services/movies_test.go

```diff
@@ -0,0 +1,36 @@
+package services
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/ashwinspg/explore-golang/dtos"
+	"github.com/ashwinspg/explore-golang/test"
+)
+
+type MovieTestSuite struct {
+	suite.Suite
+	env     test.Env
+	service *Movie
+}
+
+func (t *MovieTestSuite) SetupTest() {
+	t.T()
+	t.env = test.SetupTestEnv()
+	t.service = NewMovie(t.env.DBConn, t.env.L)
+}
+
+func (t *MovieTestSuite) TestGetMovie() {
+	movieDTO, err := t.service.GetMovie("1d3095da-3243-4a34-a7d6-2cb570446ffe")
+	t.NotNil(movieDTO)
+	t.NoError(err)
+
+	movieDTO, err = t.service.GetMovie("1d3095da-3243-4a34-a7d6-2cb570446fff")
+	t.Equal(dtos.Movie{}, movieDTO)
+	t.Error(err)
+}
```

Comment:
Can try adding test cases for the errors that have been handled in services as well.